UPGRADE-6.3.md

```diff
@@ -13,14 +13,31 @@ See the Changed Interfaces and Method Signatures section whether changes in sign
 
 The system now uses Symfony 3.4, which needs a few adjustments:
 
-- In dev mode the debug toolbar will report deprecations concerning services that are retrieved through the
-  ServiceLocator or the Symfony container directly. To be prepared for Symfony 4, ServiceLocator and container calls
-  should be used as rarely as possible (no news here) and dependency injection should be preferred. Where dependency
-  injection is not possible, services should be declared public explicitly. The deprecation warnings will also be
-  logged, potentially leading to huge log files and if there is a large number of warnings, performance in the dev
-  environment will degrade. Therefore it is recommended to deal with most of the deprecations.
-- The scope concept is gone. Remove any scope references in service definitions (e.g. Chameleon modules used
-  `scope="prototype"` in the past, which should have been changed to `shared="false"` when migrating to 6.2.x).
+In dev mode the debug toolbar will report deprecations concerning services that are retrieved through the
+ServiceLocator or the Symfony container directly. To be prepared for Symfony 4, ServiceLocator and container calls
+should be used as rarely as possible (no news here) and dependency injection should be preferred. Where dependency
+injection is not possible, services should be declared public explicitly. The deprecation warnings will also be
+logged, potentially leading to huge log files and if there is a large number of warnings, performance in the dev
+environment will degrade. Therefore it is recommended to deal with most of the deprecations.
+
+The scope concept is gone. Remove any scope references in service definitions (e.g. Chameleon modules used
```

Comment:
I wouldn't be negative here.
Just state "change to shared=false".

Reply:
Didn't mean to be negative - I changed phrasing a bit, I hope this sounds less harsh.